### PR TITLE
Fix #2238: Install clang-format 6.0

### LINKED ISSUE #2238: GitHub CI does not support clang less than 9 for Ubuntu
As of March 29th `clang-format` 6 is not found for Ubuntu 16.04, 18.04, and 20.04.

If we want to continue to support clang 6 as the minimum supported version we must install it in the CI scripts.

https://github.com/actions/virtual-environments/issues/2950

### PR DESCRIPTION
**Pull request by ekrich**
Fix for https://github.com/scala-native/scala-native/issues/2238

### CODE REVIEW

**Review by catap — APPROVED** [review]
LGTM

**Review by WojciechMazur — APPROVED** [review]

File: .github/workflows/check-lint.yml [hunk 1]
```diff
@@ -8,7 +8,11 @@ jobs:
   check-lint:
     runs-on: ubuntu-18.04
     steps:
+      - name: Install clang-format
+        run: |
+          sudo apt update
+          sudo apt install clang-format-6.0
```
[the remark this answers is not in the document]
Comment (WojciechMazur) [reply .github/workflows/check-lint.yml:14]: I've read that GitHub actions provide password-less `sudo` so it should work, however, I have never used it before. Did you have the opportunity to test is before in some other Github Actions CI?

**Review by WojciechMazur — CHANGES_REQUESTED** [review]

File: .github/workflows/check-lint.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ekrich) [reply .github/workflows/check-lint.yml:14]: I have not tried without sudo - it works now - let me check to logs. My guess it that it checks if it is running in a tty and suppresses interaction if not. Do you want me to try without sudo?

[the remark this answers is not in the document]
Comment (ekrich) [reply .github/workflows/check-lint.yml:14]: According to the docs `sudo` is needed. https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners

[the remark this answers is not in the document]
Comment (ekrich) [reply .github/workflows/check-lint.yml:14]: I actually don't see `-y` as an option for `apt`. It is an option for `apt-get`. http://manpages.ubuntu.com/manpages/trusty/en/man8/apt.8.html

[the remark this answers is not in the document]
Comment (ekrich) [reply .github/workflows/check-lint.yml:14]: I guess it seem fine as is - https://github.com/scala-native/scala-native/runs/2250741974

[the remark this answers is not in the document]
Comment (ekrich) [reply .github/workflows/check-lint.yml:14]: @WojciechMazur I think this is correct as is based on the docs.

**Review by WojciechMazur — APPROVED** [review]

**Pull request merged**